ouroboros-consensus-byron/test/Test/ThreadNet/RealPBFT.hs

```diff
@@ -81,10 +83,18 @@ import           Test.Util.Orphans.Arbitrary ()
 import           Test.Util.Shrink (andId, dropId)
 import qualified Test.Util.Stream as Stream
 
+newtype K = K SecurityParam
+  deriving (Show)
+
+instance Arbitrary K where
```

Comment:
`SecurityParam` already has an `Arbitrary` instance. Maybe give an instance for `Small SecurityParam` instead? (And can then just call `shrink`).

Reply:
Elaborated in commit 1acb1cbe2cebcc7afc8b829d9310e48fc85456c3